### ISSUE TITLE
setup.sh fails on Ubuntu 20.04 => mysql version not found

### ISSUE THREAD

**Issue by schrodinger72** [OCSInventory-NG/OCSInventory-Server#295]
Hello,

regarding these lines of code:
https://github.com/OCSInventory-NG/OCSInventory-Server/blob/824000f873071b32e24b4ede7427d0673a594d6d/setup.sh#L152-L153

on Ubuntu 20.04, `-f6` has to be changed to `-f4`:

```
mysql -V | cut -d' ' -f4 | cut -d'.' -f1
mysql -V | cut -d' ' -f4 | cut -d'.' -f2
```

Please anyone confirm?!

Thanks

**Comment by schrodinger72** [comment]
Sorry, just noticed it's fixed on master:

with commit 82e7b665f36b03eac8ab175822952cf0232c1632

**Comment by gillesdubois** [comment]
Hi @schrodinger72 ,

No problems, you might be using an old version because this fix has been integrated in the 2.8 / 2.8.1 available here : https://github.com/OCSInventory-NG/OCSInventory-ocsreports/releases/tag/2.8.1

Have a nice day !

Gilles.
